Cargo.toml

```diff
@@ -15,10 +15,12 @@ edition = "2018"
 publish = false # no accidents while in dev
 
 [dependencies]
-hyper = "=1.0.0-rc.1"
+hyper = { version = "1.0.0-rc.1", features = ["server", "http1", "http2"] }
```

Comment:
You could hide this behind a feature of `hyper-util` (so someone using some other part of `hyper-util` doesn't get forced to pick up http1, http2 and server support with all the extra dependencies that might pull in).